packages/suite/src/views/wallet/coinmarket/buy/components/BuyForm/Footer/index.tsx

```diff
@@ -94,48 +94,53 @@ const Footer = () => {
                 <Label>
                     <Translation id="TR_BUY_OFFERS_FOR" />
                 </Label>
-                <Controller
-                    control={control}
-                    defaultValue={
-                        accountHasCachedRequest && quotesRequest?.country
-                            ? {
-                                  label: regional.countriesMap.get(quotesRequest.country),
-                                  value: quotesRequest.country,
-                              }
-                            : defaultCountry
-                    }
-                    name={countrySelect}
-                    render={({ onChange, value }) => {
-                        return (
-                            <CleanSelect
-                                noTopLabel
-                                isDropdownVisible
-                                isHovered
-                                options={regional.countriesOptions}
-                                isSearchable
-                                value={value}
-                                formatOptionLabel={(option: Option) => {
-                                    const labelParts = getCountryLabelParts(option.label);
-                                    if (!labelParts) return null;
+                {/* <div> is here in order to achieve proper alignment */}
```

Comment:
What is this div for?

Reply:
The default behavior of `<Select/>` is to take 100% of available width, which aligned the dropdown more to the right than it was before this refractor. So I wrapper the _Country Select_ into a `<div>` in order to achieve proper alignment. 

But I found a more native way how to achieve this behavior by passing  `width: max-content;` to the Wrapper (see the latest changes)